packages/bridge-ui/src/relayer-api/service.ts

```diff
@@ -0,0 +1,192 @@
+import axios from 'axios';
+import { BigNumber, Contract, ethers } from 'ethers';
+import Bridge from '../constants/abi/Bridge';
+import ERC20 from '../constants/abi/ERC20';
+import TokenVault from '../constants/abi/TokenVault';
+import { chains } from '../domain/chain';
+import { MessageStatus } from '../domain/message';
+
+import type { BridgeTransaction } from '../domain/transactions';
+import { chainIdToTokenVaultAddress } from '../store/bridge';
+import { get } from 'svelte/store';
+import type { RelayerAPI, RelayerBlockInfo } from 'src/domain/relayerApi';
+
+class RelayerAPIService implements RelayerAPI {
+  private readonly providerMap: Map<number, ethers.providers.JsonRpcProvider>;
+  constructor(providerMap: Map<number, ethers.providers.JsonRpcProvider>) {
+    this.providerMap = providerMap;
+  }
+
+  async GetAllByAddress(address: string, chainID?: number): Promise<BridgeTransaction[]> {
+    if(!address) {
+     throw new Error("Address need to passed to fetch transactions");
+    }
+    let params = `address=${address}`;
+
+    if(chainID) {
+      params += `&chainID=${chainID}`;
+    }
+
+    
+    const requestURL = `${import.meta.env.VITE_RELAYER_URL}events?${params}`;
```

Comment:
1) can we use `import.meta.env.VITE_RELAYER_URL` in the constructor to set a `this.baseUrl` property? If possible we should be using as much of the env vars in `App.svelte` when initializing hte app as possible, and it also makes it more testable.

2) can we use axios's idiomatic get params here? ie:  
```ts
const requestURL = `${this.baseURL}events`;
const params = {
   address: address,
chainID: chainID
}
await axios.get(requestURL, params);
):
```

Reply:
Done